react/features/share-room/middleware.ts

```diff
@@ -53,6 +53,7 @@ function _shareRoom(roomURL: string, { dispatch, getState }: IStore) {
                 .then(
                     /* onFulfilled */ value => {
                         onFulfilled(value.action === Share.sharedAction);
+                        dispatch(toggleShareDialog(false));
```

Comment:
you also need to do that in the failure case.

Reply:
Correct! Will update.